lib/Akamai/Edgegrid.pm

```diff
@@ -221,21 +221,34 @@ sub new {
     for my $arg (@local_args) {
         $self->{$arg} = $local{$arg};
     }
-
-    for my $arg (@required_args) {
```

Comment:
I'd prefer to keep the ability to pass in args and make the config file set the passed in args.  This is how the python library is implemented.